fix: update 0021 to F00C and eth-mainnet to eth

commoddity opened the pull request:
This PR fixes the morse E2E test by updating the `0021` chain ID to `F00C` and the `eth-mainnet` alias to `eth` to match the change to the new F-chains.

Olshansk reviewed (changes requested):
The changes from `0021` to `F00C` LGTM but this PR seems to have unrelated files.

![Screenshot 2024-11-20 at 12 54 58 PM](https://github.com/user-attachments/assets/e0c2d804-795c-4a85-b16b-a1848b11f342)

commoddity commented:
> The changes from `0021` to `F00C` LGTM but this PR seems to have unrelated files.
> 
> ![Screenshot 2024-11-20 at 12 54 58 PM](https://private-user-images.githubusercontent.com/1892194/388260754-e0c2d804-795c-4a85-b16b-a1848b11f342.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3MzIxMzY3NDUsIm5iZiI6MTczMjEzNjQ0NSwicGF0aCI6Ii8xODkyMTk0LzM4ODI2MDc1NC1lMGMyZDgwNC03OTVjLTRhODUtYjE2Yi1hMTg0OGIxMWYzNDIucG5nP1gtQW16LUFsZ29yaXRobT1BV1M0LUhNQUMtU0hBMjU2JlgtQW16LUNyZWRlbnRpYWw9QUtJQVZDT0RZTFNBNTNQUUs0WkElMkYyMDI0MTEyMCUyRnVzLWVhc3QtMSUyRnMzJTJGYXdzNF9yZXF1ZXN0JlgtQW16LURhdGU9MjAyNDExMjBUMjEwMDQ1WiZYLUFtei1FeHBpcmVzPTMwMCZYLUFtei1TaWduYXR1cmU9MGY2MDNmZjgzZGEyNGUxMDZjZTJiOTlkNTE2ODFmMDAyZTQ0NTcyMGE2Y2U5MGU3YmUzMDU5OTVjYjIxZTU5YyZYLUFtei1TaWduZWRIZWFkZXJzPWhvc3QifQ.U-mUuua-2KWbZwVM2wkIrRjR1vSaFfd84qJXIh6XWoY)

Removed and updated .gitignore.

Olshansk commented:
@commoddity Reminder to update github Issue/PR metadata AND re-request a re-review once its ready.
